web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.tsx

```diff
@@ -185,18 +185,20 @@ function ProjectTopSection({
               {primaryDescription}
             </Body>
           )}
-          <Link to={`/credit-classes/${creditClassSanity?.path}`}>
-            <CreditClassCard
-              title={<BlockContent content={creditClassSanity?.nameRaw} />}
-              description={
-                <BlockContent
-                  content={creditClassSanity?.shortDescriptionRaw}
-                />
-              }
-              imgSrc={getSanityImgSrc(creditClassSanity?.image)}
-              sx={{ mt: [2, 4], py: [2, 6] }}
-            />
-          </Link>
+          {creditClassSanity && (
```

Comment:
Maybe we could `WithLoader` with the `skeleton variant to reduce layout shift a bit?

Reply:
if there's no sanity credit class, we don't want to display it the card all, so we wouldn't want to display the skeleton at all in this case

Reply:
Make sense, we could plug it to`isFetching` flag from `react-query` instead then but then we would end up showing a skeleton without data after in some cases which could be weird.
Let's leave it like that hen.